wallet/how-to/use-non-evm-networks/starknet/manage-starknet-accounts.md

```diff
@@ -0,0 +1,238 @@
+  const account = await connectStarknetAccount();
+  if (account) {
+    document.getElementById('accountAddress').innerText = `Account Address: ${account}`;
+  }
+};
+```
+
+## Switch between accounts
+
+With the `get-starknet` library, you can allow users to switch accounts by re-enabling the wallet. 
+MetaMask will prompt the user to select a different account if multiple accounts are available.
+
+```javascript
+const switchStarknetAccount = async () => {
+  try {
+    const starknet = getStarknet();
+    await starknet.enable();  // Prompts the user to select an account
+    const account = starknet.selectedAddress;
+    console.log('Switched to Starknet Account:', account);
+    
+    return account;
+  } catch (error) {
+    console.error('Error switching Starknet account:', error);
+  }
+};
+```
+
+## Manage account transactions
+
+You can manage transactions with `get-starknet`:
+
+```javascript
+
+const invokeStarknetContract = async () => {
+  try {
+    const starknet = getStarknet();
+    await starknet.enable();  // Make sure the wallet is enabled
+
+    const contractAddress = '0xYourContractAddress';  // Replace with your contract address
+    const entrypoint = 'function_name';  // The function you want to call
+    const calldata = [/* your function arguments */];  // Replace with calldata
+
+    const result = await starknet.invoke({
+      contractAddress: contractAddress,
+      entrypoint: entrypoint,
+      calldata: calldata
+    });
+
+    console.log('Transaction result:', result);
+  } catch (error) {
+    console.error('Error invoking contract:', error);
+  }
+};
+```
+
+# Comprehensive Guide to Managing Starknet Accounts
+
+## 1. Retrieving and Displaying Connected Starknet Accounts
+
+To retrieve and display connected Starknet accounts, you'll typically use the `get-starknet` library in combination with React hooks for state management.
+
+```typescript
+import { useStarknet, useConnectors } from '@starknet-react/core';
+import { useState, useEffect } from 'react';
+
+function AccountDisplay() {
+  const { account } = useStarknet();
+  const { available, connect, disconnect } = useConnectors();
+  const [accountAddress, setAccountAddress] = useState<string | undefined>();
+
+  useEffect(() => {
+    setAccountAddress(account?.address);
+  }, [account]);
+
+  return (
+    <div>
+      {accountAddress ? (
+        <p>Connected Account: {accountAddress}</p>
+      ) : (
+        <p>No account connected</p>
+      )}
+      {available.map((connector) => (
+        <button key={connector.id()} onClick={() => connect(connector)}>
+          Connect {connector.name()}
+        </button>
+      ))}
+      {account && (
+        <button onClick={() => disconnect()}>Disconnect</button>
+      )}
+    </div>
+  );
+}
+```
+
+This component will display the connected account address if available, and provide buttons to connect or disconnect accounts.
+
+## 2. Switching Between Multiple Starknet Accounts
```

Comment:
As mentioned earlier, things related to multiple starknet accounts are not yet supported in the Starknet Snap. We can remove this section for now.